run-system-tests/action.yml

```diff
@@ -197,6 +197,33 @@ runs:
         (./zwallet version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZWallet Version Info:/") || true
         (./zbox version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZBox Version Info:/") || true
 
+    - name: "Stake blobbers using above built binaries"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        echo '#!/bin/bash
+        echo
+        echo "==========================================================="
+        echo "STAKING BLOBBERS USING ZWALLETCLI & ZBOXCLI BINARIES"
+        echo "==========================================================="
+        echo
+        cd tests/cli_tests
+        cat ./config/blobber_stake_wallet.json
+        ./zwallet register --configDir ./config --config ./zbox_config.yaml --wallet ../ignore > build.log 2>&1 || { cat build.log && echo "::error title=Failed to stake the blobbers::failed to register wallet through zwalletCLI built using above branch and goSDK." && exit 1; }
+        for i in {1..6}
+        do
+          ./zwallet faucet --methodName "pour" --input "{}" --tokens 2 --configDir ./config --config ./zbox_config.yaml --wallet ../ignore > build.log 2>&1 || { cat build.log && echo "::error title=Failed to stake the blobbers::failed to execute faucet through zwalletCLI built using above branch and goSDK." && exit 1; }
+        done
+
+        blobber_ids=( $(./zwallet getblobbers --silent --configDir ./config --config ./zbox_config.yaml --wallet ../ignore | grep -o "[a-z0-9]\{64\}") )
+        for id in "${blobber_ids[@]}"
+        do
+          ./zbox sp-lock --blobber_id $id --tokens 1 --configDir ./config --config ./zbox_config.yaml --wallet ../ignore > build.log 2>&1 || { cat build.log && echo "::error title=Failed to stake the blobbers::failed to use sp-lock in zwalletCLI built using above branch and goSDK." && exit 1; }
+        done
+        echo "Stake blobbers successfully..."
+        ' > ./stake_blobbers.sh;
+        chmod 777 ./stake_blobbers.sh;
+        ./stake_blobbers.sh
```

Comment:
do we want to re-attempt this if it fails? then if all attempts fail, redeploy the network?
I know sometimes the blubber stake container restarts due to failures

Reply:
Added retries

Reply:
Resolved